paymail.go

```diff
@@ -168,13 +168,13 @@ func finalizeP2PTransaction(ctx context.Context, client paymail.ClientInterface,
 	response, err := client.SendP2PTransaction(p4.ReceiveEndpoint, p4.Alias, p4.Domain, p2pTransaction)
 	if err != nil {
 		if transaction.client != nil {
-			transaction.client.Logger().Info(ctx, fmt.Sprintf("finalizeP2PTransaction(): error %s for TxID: %s, reason: %s", p4.Format, transaction.ID, err.Error()))
+			transaction.client.Logger().Info().Msgf("finalizeP2PTransaction(): error %s for TxID: %s, reason: %s", p4.Format, transaction.ID, err.Error())
```

Comment:
finalizeP2PTransaction() can be added to Str().

Reply:
Removed function name from logs